exporter/awsemfexporter/README.md

```diff
@@ -0,0 +1,37 @@
+# AWS CloudWatch EMF Exporter for OpenTelemetry Collector
+
+This exporter converts OpenTelemetry metrics to 
+[AWS CloudWatch Embedded Metric Format(EMF)](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html)
+and then sends them directly to CloudWatch Logs using the 
+[PutLogEvents](https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_PutLogEvents.html) API.
+
+## Data Conversion
+Convert OpenTelemetry ```Int64DataPoints```, ```DoubleDataPoints```, ```SummaryDataPoints``` metrics datapoints into CloudWatch ```EMF``` structured log formats and send it to CloudWatch. Logs and Metrics will be displayed in CloudWatch console.
+
+## Exporter Configuration
+
+The following exporter configuration parameters are supported.
+
+| Name              | Description                                                            | Default |
+| :---------------- | :--------------------------------------------------------------------- | ------- |
+| `log_group_name`  | Customized log group name                                              |"/metrics/default"|
+| `log_stream_name` | Customized log stream name                                             |"otel-stream"|
+| `namespace`       | Customized CloudWatch metrics namespace                                | "default" |
+| `endpoint`        | Optionally override the default CloudWatch service endpoint.           |         |
+| `no_verify_ssl`   | Enable or disable TLS certificate verification.                        | false   |
+| `proxy_address`   | Upload Structured Logs to AWS CloudWatch through a proxy.              |         |
+| `region`          | Send Structured Logs to AWS CloudWatch in a specific region.           | determined by metadata |
+| `local_mode`      | Local mode to skip EC2 instance metadata check.                        | false   |
```

Comment:
Can we remove `local_mode`? I know our other exporter has it but I want to consider removing it there too - I had trouble explaining this flag to a user before. We use it to control returning errors, but I don't think that's actually a good idea. I think we also use it to flag whether to fetch region info, but it seems ok to just try in any case instead of having this confusing flag.

Reply:
yes. will remove this `local_mode`